department-of-veterans-affairs/va.gov-cms
# VACMS-19013: Adding Vet Center services

## Description

Relates to #19013

## Testing done
Manually

## Screenshots
<img width="1769" alt="Screenshot 2024-08-21 at 11 35 15 AM" src="https://github.com/user-attachments/assets/da1c1a79-9b1c-49ae-9f6a-f0b836a0839a">


## QA steps
As a content admin
1. Go to the [Facility services content audit](https://pr19014-ea1hvehzgnmspuepp6d2g37qkxuzjahq.ci.cms.va.gov/admin/content/audit/facility-services)
   - [x] Confirm that you can filter by "Vet Center - Facility Service" in the **Content type** exposed filter
   - [x] Choose "Vet Center - Facility Service" in the **Content type** exposed filter
   - [x] Click **Filter**
   - [x] Confirm that Vet Center - Facility Services are returned in the view
2. Then, check that the form hasn't regressed
   - [x] Choose "VAMC Facility Health Service" in the **Content type** exposed filter
   - [x] Click **Filter**
   - [x] Confirm that VAMC Facility Health Services are returned in the view



### Select Team for PR review
